tc/sgx/trusted_worker_manager/enclave_untrusted/enclave_bridge/signup.cpp

```diff
@@ -192,7 +190,218 @@ tcf_err_t tcf::enclave_api::enclave_data::CreateEnclaveData(
     }
 
     return result;
-}  // tcf::enclave_api::base::CreateSignupData
+}  // tcf::enclave_api::base::CreateEnclaveData
+
+// XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
```

Comment:
This would result in 3 different enclave_bridge (untrusted libararies) and this would eventually require 3 different Make files for different enclaves. If this is expected, i will create separate files.